meta/meta.go

```diff
@@ -55,15 +55,16 @@ var (
 //
 
 var (
-	mMetaPrefix       = []byte("m")
-	mNextGlobalIDKey  = []byte("NextGlobalID")
-	mSchemaVersionKey = []byte("SchemaVersionKey")
-	mDBs              = []byte("DBs")
-	mDBPrefix         = "DB"
-	mTablePrefix      = "Table"
-	mTableIDPrefix    = "TID"
-	mBootstrapKey     = []byte("BootstrapKey")
-	mSchemaDiffPrefix = "Diff"
+	mMetaPrefix            = []byte("m")
+	mNextGlobalIDKey       = []byte("NextGlobalID")
+	mNextGlobalServerIDKey = []byte("NextGlobalServerID")
+	mSchemaVersionKey      = []byte("SchemaVersionKey")
+	mDBs                   = []byte("DBs")
+	mDBPrefix              = "DB"
+	mTablePrefix           = "Table"
+	mTableIDPrefix         = "TID"
```

Comment:
`TID` is used to represent transaction ID in some databases. I prefer to short it to `TblID`

Reply:
? I have not changed this, `mNextGlobalServerIDKey ` was added by me.